src/app/marketplace/page.js

```
@@ -1,111 +1,159 @@
       case 'price-low':
-        filtered.sort((a, b) => (a.price || 0) - (b.price || 0));
+        tempProducts.sort((a, b) => a.price - b.price);
         break;
       case 'price-high':
-        filtered.sort((a, b) => (b.price || 0) - (a.price || 0));
-        break;
-      case 'name':
-        filtered.sort((a, b) => (a.name || '').localeCompare(b.name || ''));
+        tempProducts.sort((a, b) => b.price - a.price);
         break;
       case 'newest':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)